src/containers/PDiskPage/PDiskPage.tsx

```diff
@@ -128,6 +129,7 @@ export function PDiskPage() {
                     <Icon data={ArrowRotateLeft} />
                     {pDiskPageKeyset('restart-pdisk-button')}
                 </ButtonWithConfirmDialog>
+                <AutoRefreshControl />
```

Comment:
It's located too close to restart button. As for me, it creates association, that it somehow correlates with `PDIsk` restart, not data refecth

In my view, this control should not be on the same level as restart button. 

The same comment applicable to `VDisk` and it's evict button.

<img width="297" alt="Screenshot 2024-07-04 at 11 55 46" src="https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/bf610c18-16d0-49c4-8f21-1e9df2525986">